### ISSUE TITLE
Add signature place holder with drag and drop funtctionality to pdf

### ISSUE THREAD

**Issue by Ashok2393** [Swati4star/Images-to-PDF#660]
Hi,
 I am working on a project which includes customization of pdf like adding a signtaure place holder to pdf with drag and drop functionality is it possible to implement with this library? Please help if you have any idea, thanks in advance for your valuable time.

**Comment by Swati4star** [comment]
I don't think this is possible. We are not allowing any kind of PDF editing directly.

**Comment by Ashok2393** [comment]
Hi, 
This is not PDF editing directly, I just want to add annotations. Is it possible?

**Comment by konstzv** [comment]
You can try to work with pdf using https://github.com/TomRoush/PdfBox-Android

**Comment by Ashok2393** [comment]
Hi Konstantin,
 I tried with *PdfBox *but i can not able implemt drag and drop
functionality to signature placeholder.
Only I can able to add static signature image to pdf page.


Regards,
Ashok Patil

**Comment by konstzv** [comment]
The simplest way 
- Show pdf in pdf viewer (many options)
- Then you need to implement dragging signature (image for example) to pdf viewer view 
- Calculate the position of the image relative to pdf viewer on drop action
- Insert image in pdf via PDFBox or something similar using knowledge about the position
